README.md

```diff
@@ -82,6 +82,7 @@ In addition, at least one Cloud Proxy (also version 8.10 or later) must be set u
 * Docker 20.10.0 or later. Updating to the latest stable version is recommended. For instructions on installing Docker,
   go to [Docker's installation documentation](https://docs.docker.com/engine/install/),
   follow the instructions provided for your operating system.
+* Container registry with read and write access (see [FAQ and Troubleshooting](doc/troubleshooting_and_faq.md) for additional information).
```

Comment:
Can we link to the specific section(s) in the FAQ?

Also, I think we need to be clear about where the access is required from. We need to be able to access it from the development box _and_ any cloud proxy it will be run from.